src/modules/display.js

```diff
@@ -0,0 +1,38 @@
+import { updateLikes } from './displaylikes.js';
```

Comment:
According to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/153608236-d573bde0-e34a-4566-9133-c72f6010faa9.png)

![image](https://user-images.githubusercontent.com/57421823/153608652-3e0e34be-b554-4eca-8528-da89fc74c6ee.png)


I couldn't see a counter for the number of main items (movies in this case).  

Therefore:

- [ ] Kindly please implement a counter for the number of items presented on the home page.  Remember:  Please don't use the API to give you this number.  

Example from the wireframe:

![image](https://user-images.githubusercontent.com/57421823/153608537-fd317d03-57f7-4eb6-b3de-fcaf37504172.png)

That way, we will meet this requirement 💪

Reply:
Resolved ✔️